dali/operators/reader/loader/video/frames_decoder.cc

```diff
@@ -84,6 +84,18 @@ const std::vector<AVCodecID> FramesDecoder::SupportedCodecs = {
   AVCodecID::AV_CODEC_ID_HEVC
 };
 
+int64 FramesDecoder::NumFrames() const {
```

Comment:
As this call might fail, maybe it would be useful to have some method to check if the number of frames is available (`NumFramesAvailable` or smth like that).

Reply:
Maybe it should not ENFORCE and just return the number. If it <=0  then it is not available?

Reply:
How about we add it when we will be building a code path that uses it? For now, we are not implementing this scenario where this is not available.

Reply:
If we add the part that uses it, it would happen on the operator level, and the loader remains the same.
I think should move this enforcement to the operator, which doesn't support containers without the number of frames that are known ahead, and don't throw it here. Now it seems that this enforcement here seems to be artificial.

Reply:
Done, I changed it to retrun 0 and added a note to the doc about it.